client/blocks/plugins-update-manager/hooks/use-prepare-plugins-tooltip-info.tsx

```diff
@@ -1,16 +1,15 @@
 import { createInterpolateElement } from '@wordpress/element';
-import { SitePlugin, useSitePluginsQuery } from 'calypso/data/plugins/use-site-plugins-query';
+import { useCorePluginsQuery, type CorePlugin } from 'calypso/data/plugins/use-core-plugins-query';
 import type { SiteSlug } from 'calypso/types';
 
 export function usePreparePluginsTooltipInfo( siteSlug: SiteSlug ) {
-	const { data: pluginsData } = useSitePluginsQuery( siteSlug );
-	const plugins = pluginsData?.plugins ?? [];
+	const { data: plugins = [] } = useCorePluginsQuery( siteSlug, true, true );
 
 	const preparePluginsTooltipInfo = ( pluginsArgs: string[] ) => {
 		const pluginsList = pluginsArgs
 			.map(
 				( plugin: string ) =>
-					plugins.find( ( { name }: SitePlugin ) => name === plugin )?.display_name
+					plugins.find( ( { plugin: _plugin }: CorePlugin ) => _plugin === plugin )?.name
```

Comment:
Rather than going through the trouble of adding `.php` to all plugins in the response, could we just remove `.php` from the args list here before comparing it?

Reply:
Ups, I didn't see your comment, but I have merged it to the trunk.

There are two places in the module where we need to map the plugins with the extension. That's why I thought that it is better to have an optional param to do it in a single place.